src/Directory.Build.targets

```diff
@@ -1,7 +1,6 @@
 <!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
 <Project>
   <Import Project="$(RepoToolsetDir)Imports.targets" />  
-  <Import Project="$(RepoToolsetDir)VisualStudio.targets" Condition="'$(IsVsixProject)' == 'true'"/>
```

Comment:
Analyzer NuGet packages are built by projects under ```nuget``` root directory. We need special projects to do that since our nugets are composed of build outputs of multiple projects. 

Our VSIXes are built by projects ```src\{analyzer}\Setup``` (one such project per analyzer). The toolset automatically recognizes any project that contains ```source.extension.vsixmanifest``` as a VSIX producing projects and includes necessary msbuild targets and VSSDK packages to each such project.